lib/ProductOpener/Producers.pm

```diff
@@ -413,6 +415,22 @@ sub init_fields_columns_names_for_lang($) {
 }
 
 
+my %units_synonyms = (
+	"g" => "g",
+	"gr" => "g",
+	"grams" => "g",
+	"grammes" => "g",
+	"mg" => "mg",
+	"mcg" => "mcg",
+	"percent" => "percent",
+	"kj" => "kj",
+	"kcal" => "kcal",
+	"cal" => "kcal",
```

Comment:
cal is 1/1000 of a kcal

Reply:
This is not a conversion table, it is a list of synonyms used by producers when they transmit us data. In practice absolutely no producer uses cal (as in 1/1000 of kcal) as a unit for energy. When they have "cal" or "calories" in the header of a column, they always mean kcal. The units in this table are lowercased, so it's more for the "big Calories".

Reply:
Ok, just to make it obvious we could add a comment explaining what you wrote here.

Reply:
Good idea, I'll add it.